### DISCUSSION TITLE
E3v2 with SKR board FW

### DISCUSSION THREAD

**Discussion by smooreace** [mriscoc/Ender3V2S1#216] [General]
You have posted the pre-compiled FW for the E3V2/SKR mini, but it would be hugely beneficial to also have the associated config files! 

I'm currently converting from a basically stock running E3V2 with 4.2.7 board to either an SKR2 or SKR3/EZ. Wanting to keep my original v2 display, which I'm assuming is where youre actually gaining your key benefits in your FW. I'm running into insane troubles finding a clear layout of what's needed to run that stock display with an SKR board. Again, assuming, all of the SKR offerings will use the same pin changes to the get the same display up and running. Tossing those config files in the optional configs folder you include would be greatly appreciated!

**Comment by mriscoc** [comment]
I think that it is already posted in https://github.com/mriscoc/Marlin_Configurations

**Reply by smooreace** [reply]
How did I miss that? I looked in the config folder. Now I have to check my tracks. Thank you!

**Reply by smooreace** [reply]
I see... in the configurations folder on the current source is **Ender3V2S1-Released/configurations)/Special_Configurations.url**, which goes to a different "special config" set of info

https://github.com/mriscoc/Special_Configurations/releases
vs
https://github.com/mriscoc/Marlin_Configurations

Again, thank you!
